cpg-analysis/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/UnreachableEOGPass.kt

```diff
@@ -95,3 +96,6 @@ class UnreachableEOGPass : Pass() {
         // nothing to do
     }
 }
+
+@ExecuteBefore(ControlFlowSensitiveDFGPass::class)
+class UnreachableEOGPassBeforeDFG() : UnreachableEOGPass()
```

Comment:
Not sure I understand 100% why this is necessary

Reply:
We have to run the pass before the Control Flow Sensitive DFG Pass. However, the DFG edges can change which could result in other evaluation results if we want to run the UnreachableEOGPass again afterwards (that's also why I created a second class which is identical to the other class).